ocaml/idl/datamodel_common.ml

```diff
@@ -8,7 +8,7 @@ open Datamodel_roles
               When introducing a new release, bump the schema minor version to the next hundred
               to leave a gap for potential hotfixes needing to increment the schema version.*)
 let schema_major_vsn = 5
-let schema_minor_vsn = 300
+let schema_minor_vsn = 301
```

Comment:
Please make this 400 as the comment above indicates (this won't be part of Naples, therefore bump version by 100 to leave gap for fixes).